### PR TITLE
Itsadssd 29925 header refactor

### PR DESCRIPTION
**Pull request by jen-lui**

### CODE REVIEW

**Review by Bcrunk — APPROVED** [review]

File: packages/storybook-html/src/components/header/header_no-local-search.html [hunk 1]
```diff
@@ -1,60 +1,41 @@
...
-                <div class="nav-search__query">
-                    <span class="search-query__wrapper">
+        <div class="uq-header__search-toggle">
+            <button class="uq-header__search-toggle-button">Search</button>
+        </div>
+        <div class="uq-header__search">
+            <form>
+                <fieldset>
+                    <legend class="uq-header__search-title">What are you looking for?</legend>
+                    <div class="uq-header__search-range">
```
Comment (uqnm) [inline packages/storybook-html/src/components/header/header_no-local-search.html:14]: did you exclude the input on purpose here?

**Review by uqnm — COMMENTED** [review]
header-padding

is this the padding issue you were reffering to?

File: packages/storybook-html/src/components/header/header_no-local-search.html (hunk 1 above)
Comment (Bcrunk) [reply packages/storybook-html/src/components/header/header_no-local-search.html:14]: I don't think we need the extra text "search all uq websites"

**Comment by jen-lui** [comment]
@uqnm the padding on uq-header__container changes from 1.5rem to 1rem on mobile

File: packages/storybook-html/src/components/header/header_no-local-search.html (hunk 1 above)
Comment (jen-lui) [reply packages/storybook-html/src/components/header/header_no-local-search.html:14]: Do you mean the radio input? Shouldn't be necessary when there is no selection needed for this search.

Comment (uqnm) [reply packages/storybook-html/src/components/header/header_no-local-search.html:14]: then lets remove the label as well

Comment (jen-lui) [reply packages/storybook-html/src/components/header/header_no-local-search.html:14]: I thought it was there to let users know what sites they are searching but noticed it is not on UQ home page. Happy to remove, any thoughts @Bcrunk ?

Comment (jen-lui) [reply packages/storybook-html/src/components/header/header_no-local-search.html]: pushed up a commit with label removed

**Review by uqnm — APPROVED** [review]

**Pull request merged**